src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,142 @@
+            throw new DataProcessingException("Can't get manufacturer by id, with id:" + id, e);
+        }
+        return Optional.ofNullable(manufacturer);
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> manufacturerList = new ArrayList<>();
+        String sqlRequest = "SELECT id, manufacturers_name, manufacturers_country "
+                + "FROM manufacturers WHERE is_deleted = false;";
+        try (
+                Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement manufacturersGetAllStatement =
+                        connection.prepareStatement(sqlRequest)
+        ) {
+            ResultSet resultSetFromDb = manufacturersGetAllStatement.executeQuery();
+            while (resultSetFromDb.next()) {
+                Manufacturer manufacturer = getDataFromResultSet(resultSetFromDb);;
+                manufacturerList.add(manufacturer);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all manufacturer", e);
+        }
+        return manufacturerList;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String sqlRequest = "UPDATE manufacturers "
+                + "SET manufacturers_name = ?, manufacturers_country = ? "
+                + "WHERE id = ? AND is_deleted = false;";
+        try (
+                Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement manufacturerUpdateStatement =
+                        connection.prepareStatement(sqlRequest)
+        ) {
+            manufacturerUpdateStatement
+                    .setString(FIRST_PARAMETER_INDEX, manufacturer.getName());
+            manufacturerUpdateStatement
+                    .setString(SECOND_PARAMETER_INDEX, manufacturer.getCountry());
+            manufacturerUpdateStatement
+                    .setObject(THIRD_PARAMETER_INDEX, manufacturer.getId());
+            manufacturerUpdateStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't do the update " + manufacturer, e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String sqlRequest = "UPDATE manufacturers "
+                + "SET is_deleted = true WHERE id = ?;";
+        try (
+                Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement manufacturerDeleteStatement =
+                        connection.prepareStatement(sqlRequest)
+        ) {
+            manufacturerDeleteStatement.setObject(FIRST_PARAMETER_INDEX, id);
+            return manufacturerDeleteStatement.executeUpdate() >= 1;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete manufacturer with id: " + id, e);
+        }
+    }
+
+    private Manufacturer getDataFromResultSet(ResultSet resultSet) {
```

Comment:
```suggestion
    private Manufacturer getManufacturerFromResultSet(ResultSet resultSet) {
```

I think it's more informative that way, you may have another similar methods in the future